modules/website/src/components/HomepageFeatures.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './HomepageFeatures.module.css';
+
+const FeatureList = [
+  {
+    title: 'Contract First',
+    Svg: require('../../static/img/undraw_sync_files.svg').default,
+    description: (
+      <>
+        Smithy4s generates protocol-agnostic Scala code from Smithy, a concise, readable, language-agnostic
+        format
```

Comment:
```suggestion
        format.
```

just for consistency with other blocks